tests/installation/partitioning_raid.pm

```diff
@@ -56,8 +56,16 @@ sub addraid($;$) {
     }
     send_key $cmd{"next"};
     assert_screen 'partition-role', 6;
-    send_key "alt-o";    # Operating System
+    if ($step == 3 and get_var("LVM"))
+    {
+      send_key "alt-a";    # Raw Volume
```

Comment:
please use the perl style used in the rest of the file instead of placing { here